test/propagatortest.jl

```diff
@@ -0,0 +1,33 @@
+using Base.Test
+
+# Initializing the system
+
+# Hamiltonian of the system
+hamiltonian = sigmax
+# Initial State
+init_state = QuArray([0.5+0.1im, 0.2+0.2im])
```

Comment:
It is better to use a normalized initial state - while being correct, everything else is just confusing.